### PR TITLE
hazelcast@5.X.rb, add hz-cli to path

### PR DESCRIPTION
**Pull request by fbarotov**

### CODE REVIEW

**Review by alparslanavci — COMMENTED** [review]
Thanks for the PR, @fbarotov. This only includes `hz-cli` to the path, is there any way to include all scripts in `bin` directory?

**Comment by fbarotov** [comment]
@alparslanavci so we should include all scripts from `bin/` to PATH right? Because from the following https://hazelcast.slack.com/archives/C026A3BMTQD/p1626352306072800?thread_ts=1626337115.067000&cid=C026A3BMTQD I got that we need only `hz` and `hz-cli` and others can be on the path but preferred not to. 

@Holmistr what do you say?

**Comment by Holmistr** [comment]
@fbarotov @alparslanavci After discussion with Can, please put all the scripts in the `/bin` directory on the PATH.

**Comment by alparslanavci** [comment]
Please see the issue that needs to be fixed before this issue: https://github.com/hazelcast/hazelcast/issues/19135

File: hazelcast@5.X.rb [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    depends_on "openjdk" => :recommended
+  
+    def install
+      libexec.install Dir["*"]
+      Dir["#{libexec}/bin/hz*"].each do |path|
+        executable_name = File.basename(path)
+        if executable_name.end_with? ".bat"
+          next
+        end
+        (bin/executable_name).write_env_script libexec/"bin/#{executable_name}", Language::Java.overridable_java_home_env
```
Comment (frant-hartm) [inline hazelcast@5.X.rb:18]: I have zero Homebrew knowledge, so I have no idea how this works exactly, but - do we need the same as for deb/rpm packages - to remove the links during uninstall?

**Review by frant-hartm — APPROVED** [review]

File: hazelcast@5.X.rb (hunk 1 above)
Comment (fbarotov) [reply hazelcast@5.X.rb:18]: No we don't need to explicitly remove symlinks, they are removed automatically by the Homebrew

**Review by frant-hartm — APPROVED** [review]

**Pull request merged**